src/entry/webtriggers/gitlab-event-handlers/handle-deployment-event.test.ts

```diff
@@ -37,25 +35,24 @@ jest.mock('../../../utils/has-deployment-after-28days');
 const mockedGetEnvironments = mocked(getEnvironments);
 const mockedGetDeployment = mocked(getDeployment);
 const mockedSendEventsToCompass = mocked(sendEventToCompass);
-const mockedGetRecentDeployments = mocked(getRecentDeployments);
-const mockedInsertMetricValues = mocked(insertMetricValues);
-const mockedHasDeploymentAfter28Days = mocked(hasDeploymentAfter28Days);
```

Comment:
None of these calls are in the current `handle-deployment-event` path. Removed them from the tests